modules/core/src/main/java/org/apache/fluo/core/oracle/OracleServer.java

```diff
@@ -195,11 +195,16 @@ public OracleServer(Environment env) throws Exception {
 
   private void allocateTimestamp() throws Exception {
     Stat stat = new Stat();
+    // FLUO-1000
+    while (!curatorFramework.getState().equals(CuratorFrameworkState.STARTED)) {
```

Comment:
In 9b5277c9c79bb3a52bb879af591d1c51ae9476cc I changed the order in which things started.  I started curator before starting the thrift server.  Given those changes, this is probably no longer needed.